app/indices/account_index.rb

```diff
@@ -13,7 +15,10 @@
   indexes users.id,                            as: :user_id
 
   set_property field_weights: { name: 2 }
-  set_property group_concat_max_len: 32_000 if System::Database.mysql?
+
+  if System::Database.mysql? # rubocop:disable Style/IfUnlessModifier
```

Comment:
If I do it inline, CircleCI fails with: syntax error near unexpected token

Reply:
even with parenthesis?

Reply:
Sorry, I've just seen this comment. I haven't tried that so I will try in another PR 😉

Reply:
https://github.com/3scale/porta/pull/592 @hallelujah